flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/functions/sql/FlinkSqlOperatorTable.java

```diff
@@ -574,9 +574,9 @@ public SqlMonotonicity getMonotonicity(SqlOperatorBinding call) {
 	public static final SqlFunction TO_BASE64 = new SqlFunction(
 		"TO_BASE64",
 		SqlKind.OTHER_FUNCTION,
-		VARCHAR_2000_NULLABLE,
+		ReturnTypes.cascade(ReturnTypes.explicit(SqlTypeName.VARCHAR), SqlTypeTransforms.TO_NULLABLE),
 		null,
-		OperandTypes.ANY,
+		OperandTypes.family(SqlTypeFamily.STRING),
```

Comment:
Thanks for your contribution~
Could you add a test for this, like the one you described in jira?

Reply:
You can add tests to `org.apache.flink.table.expressions.ScalarFunctionsTest#testToBase64`。

Reply:
Thank you @wuchong @libenchao.  We change  from OperandTypes.ANY to OperandTypes.family(SqlTypeFamily.STRING). If user apply int type to to_base function. For the before, it will throw runtimeException. And for now, it will fail in validation.
So, if we should add a test, I think we can add it to org.apache.flink.table.planner.expressions.ScalarFunctionsTest to verify fail in validation stage. For the  org.apache.flink.table.expressions#testToBase64, it already has test to verify the correct arguments.
It is right?

Reply:
You are right. I messed it with another fromBase64 PR.

Reply:
👌, Updated.